### ISSUE TITLE
Google Docs Gruppe

### ISSUE THREAD

**Issue by d135-1r43** [Vereinsfin/vereinsfin#13]
#Story 

Als Team haben wir eine Gruppe in Google Docs, um Dokumente bzw. Ordner von Dokumenten mit allen privat zu teilen.

**Comment by d135-1r43** [comment]
Hier war jetzt @eliashehme dran, oder?

**Comment by eliashehme** [comment]
Nope @sk2675 oder? - ich wars auf jeden Fall nicht

**Comment by eliashehme** [comment]
Nein, ich habe nach den Kontaktdaten für die Teilnehmerliste und danach gefragt, wer eine Lizenz für den Figma-Account benötigt.

**Comment by d135-1r43** [comment]
Okay, dann solltet ihr beide euch hier abstimmen.

**Comment by sk2675** [comment]
Habe jetzt ein Google-Drive "Struktur" angelegt. Dabei einfach Ordner oder Inahlt umschreiben, wie es euch passt.

[https://drive.google.com/drive/folders/1E8CYFOCSUzicqaBtY77BXi_ZCxOmau9Z?usp=sharing](url)

**Comment by d135-1r43** [comment]
Passt. Sind da alle Mitarbeitenden drauf berechtig?
